### PR TITLE
Introduce AssignmentsE2ETests with DataSeeding

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-15858
affects: Teacher, Student, Parent
release note: none

test plan:

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/110586197/p/66c839153504ca3821e23ca07aea0584">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/110586203/p/87bf737b28454946a166563fcc4cc90a">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/110586190/p/c07ac5fcf10d28141c7cd1d50dd981db">
    
  </a>
</details>

**Comment by davidvarga93** [comment]
Screenshot 2022-01-28 at 12 15 49

File: TestsFoundation/TestsFoundation/DataSeeding/Assignment/CreateDSAssignmentRequest.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+}
+
+extension CreateDSAssignmentRequest {
+    public struct RequestDSAssignment: Encodable {
+        let name: String
+        let description: String?
+        let published: Bool
+
+        public init(name: String = "kaki", description: String? = nil, published: Bool = true) {
```
Comment (balintbartok) [inline TestsFoundation/TestsFoundation/DataSeeding/Assignment/CreateDSAssignmentRequest.swift:41]: can we change the default value to empty or something meaningful?

Comment (davidvarga93) [reply TestsFoundation/TestsFoundation/DataSeeding/Assignment/CreateDSAssignmentRequest.swift]: Whoops

**Review by vargaat — CHANGES_REQUESTED** [review]

**Review by vargaat — APPROVED** [review]

**Review by balintbartok — APPROVED** [review]

**Pull request merged**